scc/src/main.rs

```diff
@@ -49,18 +60,46 @@ fn main() -> Result<(), Error> {
     Ok(())
 }
 
-fn setup_logger(cache_dir: &Path) -> Result<(), Error> {
+pub fn exists(file : &Path) -> bool {
+    fs::metadata(file).is_ok()
+}
+
+fn setup_logger(cache_dir: &Path, date_time_logger : bool) -> Result<(), Error> 
+{
+    let parent_dir  = cache_dir.parent().unwrap();
+    let mut log_name : String = "redscript".to_owned();
+    
+    if date_time_logger == true
+    {
+        log_name.push_str("_");
+        let time_create = OffsetDateTime::now_local().unwrap().format(&Rfc3339Format).unwrap();
+        log_name.push_str(&time_create);
+
+        log_name = log_name.replace("-", ".").replace(":", "-")[..29].to_string();
+        log_name.replace_range(20..21, "_");
+    }
+    
+    log_name.push_str(".log");
+
+    // Log directory make
+    let log_dir = parent_dir.join("logs");
+
+    if !exists(&log_dir)
+    {
+        fs::create_dir(log_dir)?;
+    }
+
     fern::Dispatch::new()
-        .format(move |out, message, rec| {
+        .format(move |out, message, rec| 
+        {
             let time = OffsetDateTime::now_local().unwrap().format(&Rfc3339Format).unwrap();
-            out.finish(format_args!("{} [{}] {}", time, rec.level(), message));
+                out.finish(format_args!("{} [{}] {}", time, rec.level(), message));
         })
         .level(log::LevelFilter::Info)
         .chain(io::stdout())
-        .chain(fern::log_file(cache_dir.join("redscript.log"))?)
+        .chain(fern::log_file(parent_dir.join("logs").join(log_name))?)
```

Comment:
change `parent_dir.join("logs")` to `log_dir`

Reply:
IDK what does he want 
![image](https://user-images.githubusercontent.com/13867290/179428194-2f920e33-a421-4632-9d9d-6cd1ad790775.png)